tests/memarray/main.c

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (C) 2017 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       Simple memarray module tests
+ *
+ * @author      Tobias Heider <t.heider@campus.lmu.de>
+ * @author      Raul Fuentes <raul.fuentes-samaniego@inria.fr>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "memarray.h"
+
+#define MAX_NUMBER_BLOCKS 10
```

Comment:
align indention of defines, add parentheses and consider making the unsigned, i.e. like 
```
#define MAX_NUMBER_BLOCKS   (10U)
#define MESSAGE_SIZE        (8U)
#define NUMBER_OF_TESTS     (12U)
```